lib/charts.js

```diff
@@ -134,26 +132,25 @@ async function renderChartJs(
     // The chart could contain Javascript - run it in a VM.
     try {
       const { getGradientFill, getGradientFillHelper } = getGradientFunctions(width, height);
-      const vm = new NodeVM({
-        console: 'off',
-        eval: false,
-        wasm: false,
-        sandbox: {
-          getGradientFill,
-          getGradientFillHelper,
-          pattern: {
-            draw: patternDraw,
-          },
-          Chart: getChartJsForVersion(version),
-        },
-      });
-      chart = vm.run(`module.exports = ${untrustedChart}`);
+      const chartFunction = new Function(
```

Comment:
Hi guys. I noted that was changed in that version. There are some reason? Is due deprecation of vm2?